### PR TITLE
Enable opening spec v3

### PR DESCRIPTION
**Pull request by constantinpape**
Enable opening files with multiscale spec v3, pending https://github.com/ome/ome-zarr-py/pull/89.

### CODE REVIEW

**Comment by joshmoore** [comment]
cc: @will-moore

File: napari_ome_zarr/_reader.py [hunk 1]
```diff
@@ -50,7 +50,18 @@ def f(*args: Any, **kwargs: Any) -> List[LayerData]:
 
         for node in nodes:
             data: List[Any] = node.data
+
+            # check if this node has a multiscale spec and try to load the axes
+            # NOTE this was the best way I found for getting the axes metadata, which is still very cumbersome
+            axes = None
+            for spec in node.specs:
+                axes = spec.lookup('multiscales', [{}])[0].get('axes', None)
```
Comment (joshmoore) [inline napari_ome_zarr/_reader.py:58]: `isinstance(spec, Multiscales)` would be one possible first test, but then we would need to add this as a helper.

An alternative would be `spec.find(Multiscales)` though I've held off adding a formal API until we get the major metadata revisions taken care of.

p.s. ... or would we even work toward `spec.find(Axes)`?

**Review by joshmoore — COMMENTED** [review]
Generally :+1: with some thoughts about the API we can work toward.

**Comment by constantinpape** [comment]
Closed in favor of #9

**Pull request closed without merging**